openapi/social/social.yaml

```
@@ -275,21 +361,57 @@ components:
             messageText:
               type: string
               description: 'The text of the message you’re making. This text is subject to limitations depending on what service you are posting to, such as Tweet length for Twitter. '
-            mediaUrls:
-              description: 'Links to pieces of media you want on the message you’re sending. These can be images, videos, and gifs. Media are subject to limitations depending on the social network you are posting to.'
+            status:
+              type: string
+              enum:
+                - scheduled
+                - posted
+              description: 'A string representing the tag assigned to a post.'
+              example: 'SCHEDULED'
```

I'm not sure why the description mentions "tag". 

Can we eliminate the `errorOccured` attribute by including it as a status? 

```suggestion
              enum:
                - scheduled
                - posted
                - errorPosting
              description: |-
                The status of the message will be automatically set to one of the following by the system:
                - `scheduled`: The message has been approved and is scheduled to be sent at the time indicated in `sendAt`
                - `posted`: The message was successfully sent to the social profile
                - `errorPosting`: An error occurred when sending the message. You can view the details in the `error` attribute.
              example: 'posted'
              readOnly: true
```